snet_cli/arguments.py

```diff
@@ -170,7 +173,9 @@ def add_agent_factory_options(parser):
     create_record_p.add_argument("contract_named_input_price", type=type_converter("uint256"), metavar="PRICE",
                                  help="desired initial job price for agent")
     create_record_p.add_argument("contract_named_input_endpoint", type=type_converter("string"), metavar="ENDPOINT",
-                                 help="initial endpoint for agebt's daemon")
+                                 help="initial endpoint for agent's daemon")
+    create_record_p.add_argument("contract_named_input_metadataURI", type=type_converter("string"), metavar="METADATA_URI",
+                                 help="uri where agent metadata is stored")
```

Comment:
`service metadata` like in the documentation? (I'm taking issue with the "agent" word here not the whole sentence)

Reply:
super small but you fixed the other one and not this one

Reply:
Fixed